11/collection-meta.yaml

```diff
@@ -252,8 +252,6 @@ collections:
     repository: https://github.com/fortinet-ansible-dev/ansible-galaxy-fortimanager-collection
   fortinet.fortios:
```

Comment:
You also need to update 11/ansible.in.

Reply:
Like [this](https://github.com/ansible-community/ansible-build-data/pull/437/commits/155ea8c45652a9ad0b83c56776d2e901b235a33e)?